content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -274,6 +275,7 @@ The parameter resource could be as following:
 
 <!--
 This policy parameter resource limits deployments to a max of 3 replicas.
+
```

Comment:
Do we also need to add a blank line to the zh para?

Reply:
It should be fine.